rs_utils.py

```diff
@@ -82,6 +84,324 @@ def close(self):
                 self.dict = None
 
 
+class RSNamespace(argparse.Namespace):
+    def __getitem__(self, key):
+        try:
+            return self.__getattribute__(key)
+        except AttributeError:
+            raise KeyError
+
+    def __setitem__(self, key, value):
+        self.__setattr__(key, value)
+
+
+class RSArgumentParser(argparse.ArgumentParser):
+
+    def __init__(self, add_help=False, ignore_unrecognized_arguments=True, **kwargs):
+        self.ignore_unrecognized_arguments = ignore_unrecognized_arguments
+        super().__init__(self, add_help=add_help, **kwargs)
+
+    def exit(self, status=0, message=None):
+        raise CommandSyntaxError(message)
+
+    def error(self, message):
```

Comment:
Remove copy-pasted docstring and commented code

Reply:
Done.